pkg/controller/catalogsourceconfig/catalogsourceconfig_controller.go

```diff
@@ -81,6 +82,8 @@ type ReconcileCatalogSourceConfig struct {
 // Result.Requeue is true, otherwise upon completion it will remove the work from the queue.
 func (r *ReconcileCatalogSourceConfig) Reconcile(request reconcile.Request) (reconcile.Result, error) {
 	log.Printf("Reconciling CatalogSourceConfig %s/%s\n", request.Namespace, request.Name)
+	// Reconcile kicked off, message Sync Channel
+	status.SendSyncMessage(nil)
```

Comment:
Shouldn't you also send a message after a successful or failed reconciliation?

Reply:
I think we just need to do it once, after the `handle` function is called.
```
err = r.CatalogSourceConfigHandler.Handle(context.TODO(), instance)
status.SendSyncMessage(err)
```
It covers both scenarios.

Reply:
I don't think we want to handle this case as an error. At this point we report failing only if we have a critical error. If we reach the failing state our operator does not recover.

Reply:
>I think we just need to do it once, after the handle function is called.

@tkashem We can't do it just once as we want to differentiate between a sync and a successful sync.

Reply:
Per our discussion this morning, I have updated the code in the latest commit to report a sync event and a sync failure if any events occur:
```
func (r *ReconcileOperatorSource) Reconcile(request reconcile.Request) (reconcile.Result, error) {
	log.Printf("Reconciling OperatorSource %s/%s\n", request.Namespace, request.Name)
	// Reconcile kicked off, message Sync Channel with sync event
	status.SendSyncMessage(nil)

	// Message Sync Channel with Failure if err is not nil at the end of this function
	var err error
	defer func() {
		if err != nil {
			status.SendSyncMessage(err)
		}
	}()
...
...
...
```